llama-index-integrations/readers/llama-index-readers-file/llama_index/readers/file/markdown/base.py

```diff
@@ -71,12 +71,12 @@ def markdown_to_tups(self, markdown_text: str) -> List[Tuple[Optional[str], str]
         return markdown_tups
 
     def remove_images(self, content: str) -> str:
-        """Get a dictionary of a markdown file from its path."""
+        """Remove images in markdown content"""
```

Comment:
This ending with a period will fix linting (same with the other one)

Reply:
oops, fixed